src/UI/Button.elm

```diff
@@ -507,15 +506,15 @@ hyperlinkView :
 hyperlinkView cfg size width body action =
     let
         attrs =
-            [ buttonWidth width
-            , Palette.color tonePrimary brightnessMiddle
-                |> Palette.toElementColor
-                |> Font.color
-            , Font.regular
-            , Font.underline
-            , Element.pointer
-            ]
-                ++ (ARIA.toElementAttributes <| ARIA.roleButton)
+            buttonWidth width
+                :: (Palette.color tonePrimary brightnessMiddle
```

Comment:
`O(n=6)` vs `6*O(1)` :joy: 
Not sure why it suggests this, maybe it's for the case we do something like `((x ++ y) ++ z)` :man_shrugging: